### ISSUE TITLE
[HOLD on 447024] [$250] The "Mark as Cash" button is displayed instead of the "Approve" button on a report submitted for approval

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#54122]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** Needs Reproduction
**Reproducible in production?:** Needs Reproduction
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @puneetlath 
**Slack conversation** (hyperlinked to channel name): [expensify_bugs](https://expensify.slack.com/archives/C049HHMV9SM/p1734105339015599)

## Action Performed:

1. A Card transaction expense is submitted by a user for approval.
2. Open the report as the approver.
3. Observe the actions available at the top of the report.

## Expected Result:
The "Approve" button should be available as the primary or secondary action, allowing the approver to approve the report.
## Actual Result:
The "Mark as Cash" button is displayed instead of the "Approve" button. The "Approve" action is not available, which is unexpected 
for a submitted report. The action button on the Reports page is also different than the one within the report
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

![image (2)](https://github.com/user-attachments/assets/ed5a1a3e-0e3b-4425-a196-c83667b92bfa)

![image (3)](https://github.com/user-attachments/assets/876bae56-9c9f-42fc-89dd-a9b8c7d2dd9d)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021869460044615032430</li>
        <li>Upwork Job ID: 1869460044615032430</li>
        <li>Last Price Increase: 2024-12-18</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @ZhenjaHorbach</details>

**Comment by puneetlath** [comment]
@garrettmknight should we open this up? Or are we handling this as part of the action buttons project?

**Comment by sakluger** [comment]
@garrettmknight Let me know if I should add the external label or if we are handling internally.

**Comment by garrettmknight** [comment]
I just encountered this while testing. Let's go `External`.

**Comment by garrettmknight** [comment]
I think we should fix this now rather than wait for action buttons.

**Comment by garrettmknight** [comment]
Still open, will make sure it gets fixed as part of the Action Buttons project.

**Comment by garrettmknight** [comment]
Same as above

**Comment by garrettmknight** [comment]
Fine eroding, will confirm when action buttons deploy.

**Comment by dylanexpensify** [comment]
How's this one looking? Still seeing?

**Comment by garrettmknight** [comment]
Yeah, this is persistent, it should go away when we implement the new primary/secondary buttons.

**Comment by garrettmknight** [comment]
Still holding on https://github.com/Expensify/Expensify/issues/447024

**Comment by garrettmknight** [comment]
Still holding on https://github.com/Expensify/Expensify/issues/447024

**Comment by garrettmknight** [comment]
@dylanexpensify this should be good to test!

**Comment by garrettmknight** [comment]
Ooops, wrong issue.

**Comment by garrettmknight** [comment]
I did test this one today and it worked though, so closing.
